lib/jsdom/living/node.js

```diff
@@ -191,3 +191,25 @@ module.exports.listOfElementsWithNamespaceAndLocalName = (namespace, localName,
     } });
   });
 };
+
+// true for all node types except document ones
+function isNode(node) {
+  return node && (
+    node.nodeType <= NODE_TYPE.COMMENT_NODE ||
+    node.nodeType === NODE_TYPE.DOCUMENT_FRAGMENT_NODE
+  );
+}
+
+// create a fragment (or just return a node for one item)
+exports.createNode = (document, nodes) => {
```

Comment:
Ok, well the spec says either string or Node, but in chrome it allows element.after(3) for example, checking only typeof node === 'string' is easier, so I can do that indeed